K2Bridge/KustoDAL/IKustoDataAccess.cs

```diff
@@ -25,5 +25,12 @@ public interface IKustoDataAccess
         /// <param name="indexName">Index name.</param>
         /// <returns>A list of Indexes.</returns>
         Task<IndexListResponseElement> GetIndexListAsync(string indexName);
```

Comment:
This method can be removed from the interface and the implementation

Reply:
Removed.